README.md

```diff
@@ -1 +1 @@
-testing semantic versioning 4.0.0
+testing semantic versioning 4.1.0
```

Comment:
- The version number in the code has been updated from 4.0.0 to 4.1.0. No further comments or suggestions needed.